### ISSUE TITLE
Refetch token metadata after baseURI update

### ISSUE THREAD

**Issue by sl1depengwyn** [blockscout/blockscout#7670]
Now after baseURI update Blockscout do bot update toke's metadata.
Possible ways to solve it:
- Add env var for tokens contract address hashes that should be refetched after restart
- Add some detection of most common setURI methods and refetch metadata automatically

**Comment by Lizard-King101** [comment]
where are we at with this issue?

**Comment by SurenFTN** [comment]
@vbaranov is there any functional on the version 6.4.0 or higher to update metadatas

**Comment by danirabbani90** [comment]
I dont think so the following issue has been resolved i am still facing this issue when i try to refetch metadata even though the recaptcha is disabled it constantly asks for recaptcha someone needs to look into this.
